### PR TITLE
Update standalone dockerfile and docs

### PR DESCRIPTION
**Pull request by vprithvi**
Todo: integration tests for http endpoints

### CODE REVIEW

File: cmd/standalone/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,22 @@
 FROM scratch
-# expose agent UDP and sampling ports and query port
-EXPOSE 5775/udp 6831/udp 6832/udp 5778 16686
+
+# Agent zipkin.thrift
```
Comment (black-adder) [inline cmd/standalone/Dockerfile]: add "compact" for consistency

File: cmd/standalone/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+EXPOSE 5775/udp
+
+# Agent jaeger.thrift compact
+EXPOSE 6831/udp
+
+# Agent jaeger.thrift binary
+EXPOSE 6832/udp
+
+# Sampling processor
```
Comment (black-adder) [inline cmd/standalone/Dockerfile]: agent config HTTP endpoint

File: docs/getting_started.md [hunk 3]
```diff
@@ -7,10 +7,20 @@ This image, designed for quick local testing, launches the Jaeger UI, collector,
...
 
-You can then navigate to `http://localhost:16686` to access the Jaeger UI. 
+You can then navigate to `http://localhost:16686` to access the Jaeger UI.
+
+Port | Protocol | Function
+---- | -------  | ---
+5775 | UDP      | zipkin.thrift agent using compact thrift protocol
+6831 | UDP      | jaeger.thrift agent using compact thrift protocol
+6832 | UDP      | jaeger.thrift agent using binary thrift protocol
+5778 | HTTP     | sampling service
```
Comment (black-adder) [inline docs/getting_started.md]: same as above

File: docs/getting_started.md [hunk 4]
```diff
@@ -7,10 +7,20 @@ This image, designed for quick local testing, launches the Jaeger UI, collector,
...
 
 ```bash
-docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 -p16686:16686 jaegertracing/all-in-one:latest
+docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 -p16686:16686 -p14268:14268 jaegertracing/all-in-one:latest
 ```
 
-You can then navigate to `http://localhost:16686` to access the Jaeger UI. 
+You can then navigate to `http://localhost:16686` to access the Jaeger UI.
+
+Port | Protocol | Component | Function
```
Comment (yurishkuro) [inline docs/getting_started.md]: Add a sentence explaining this table, it's a bit sudden.

File: cmd/standalone/Dockerfile [hunk 5]
```diff
@@ -1,6 +1,22 @@
 FROM scratch
-# expose agent UDP and sampling ports and query port
-EXPOSE 5775/udp 6831/udp 6832/udp 5778 16686
+
+# Agent zipkin.thrift compact
```
Comment (yurishkuro) [inline cmd/standalone/Dockerfile]: 🥇

File: docs/getting_started.md (hunk 4 above)
Comment (yurishkuro) [inline docs/getting_started.md]: Maybe even a subsection header

Comment (vprithvi) [inline docs/getting_started.md]: Agree, also the docker command itself isn't readable in it's current state. Perhaps we can refactor the deployment page, and link to that instead. I'll give it some thought tonight/tomorrow.

**Pull request merged**